tests/integration/backend_test.py

```diff
@@ -374,7 +394,7 @@ def test_classical(
     b = authenticated_quum_backend
 
     c = b.get_compiled_circuit(c)
-    assert b.run_circuit(c, n_shots=10).get_counts()
+    assert b.run_circuit(c, n_shots=10, language=language).get_counts()  # type: ignore
```

Comment:
Not sure why we need these `type:ignore` now; is `language` not properly typed?

Reply:
It is because all the parameters here are not individually defined, they are handled as dict:
```tests/integration/backend_test.py:405: error: Argument "language" to "run_circuit" of "Backend" has incompatible type "Language"; expected "Union[int, float, str, None]"  [arg-type]```

run_circuit is not defined in the backend, so we can't change this